content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,52 @@
+# Machine Learning Model Registries
+
+A **model registry** is a central catalog for data artifacts produced in
+multiple data science projects. Models can be discovered, audited, shared,
+tested, and deployed from here. DVC works on top of Git to enable these
+capabilities on your existing ecosystem. And other tools from
+[Iterative](https://iterative.ai/) can provide further advanced features and
+better usability as well.
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
+[data registries]: /doc/use-cases/data-registries
+
+DVC model registries give your team key collaborative capabilities:
+
+- Store, organize, and track the [versions] of different models effectively.
+- Full data provenance and lineage trails guarantee that your modeling is
+  reproducible.
+- Share and control access to early-stage [experiments] and production-ready
```

Comment:
this is not clear to be honest

Reply:
Updated to:

> - Share and control access to any data artifact, whether these are early-stage [experiments] or production-ready models.

I think that's clearer. I'm just not sure it's necessary to mention that range of possibilities anymore... but it's nice to link to experiments from somewhere here. WDYT?

Reply:
okay, got it, that's fine!